src/main/java/com/db/crud/voting/service/impl/AgendaServiceImpl.java

```diff
@@ -98,6 +108,10 @@ private void authenticateUserAdmin(User user) {
         }
     }
```

Comment:
```suggestion
    private void authenticateUserAdmin(User user) {
        if (user.getUserType() != UserType.ADMIN) {
            log.error("User not Authenticated: "+user);
            throw new AuthorizationException("You don't have authorization to create a agenda!");
        }
    }
```

Esse método deveria pertencer ao service de Usuários. O service de usuário é responsável por todas as operações relacionadas ao gerenciamento e autenticação de usuários!

Reply:
Sim, esse método foi separado do createAgenda e eu não me toquei que ele poderia ser colocado no User e ser chamado apenas a userService no lugar dele.

Essa mudança também foi feita nesse commit: [move LogObj to logService and authorizeAdmin to userService](https://github.com/AurorinhaBoreal/VotacaoCRUD-BackEnd/pull/37/commits/e548d5696fe68876cebb85321e63d89fb196b58b)